From 79c6861834cab9a887070da19e9adcfe1f1c24c5 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 19 Jul 2018 17:43:19 +0800
Subject: [PATCH 276/706] drm: hpd: Increase the max Lane link rate to 5.4Gb/s

This patch increases the max Lane link rate to 5.4Gb/s.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 3bbd7b498e97a9ee6c3ce35aee0c28a0c23b33ee)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index faf2ed62369f..2a7b27e72e54 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -1001,7 +1001,7 @@ static int imx_hdp_imx_bind(struct device *dev, struct device *master,
 	}
 #endif
 
-	hdp->link_rate = AFE_LINK_RATE_2_7;
+	hdp->link_rate = AFE_LINK_RATE_5_4;
 
 	hdp->dual_mode = false;
 
-- 
2.17.1

